src/styles/main.scss

```diff
@@ -299,6 +299,27 @@ div.align-center img{
 	}
 }
 
+/* highlightjs number block of numbers */
+table tr td.hljs-ln-numbers {
+    -webkit-touch-callout: none;
+    -webkit-user-select: none;
+    -khtml-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
```

Comment:
is this to prevent users from being able to select the numbers in all different browsers?

Reply:
Yes! https://developer.mozilla.org/en-US/docs/Web/CSS/user-select